README.md

```diff
@@ -1,2 +1,53 @@
-# Portfolio
-My portfolio
+# Project Name
+Portfolio: set up and mobile first
+
+
+## Built With
+HTML
+CSS
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
```

Comment:
- Kindly try to customize your readme file as much as possible with the available information. If a field in the readme file is not available, kindly remove it. Provide steps that users can follow to use the project on their local machines. The MIT link is also broken. Kindly fix it :+1: